src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetSecureRandomCommand.cs

```diff
@@ -0,0 +1,717 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
```

Comment:
Can we add `#nullable enable`?

Reply:
That creates a bunch of build issues as the current code doesn't support it and would require much more changes than intended with this PR